test_runner/build.gradle.kts

```diff
@@ -378,4 +378,8 @@ val resolveArtifacts by tasks.registering {
     }
 }
 
-tasks.test { dependsOn(resolveArtifacts) }
+tasks.test {
+    maxHeapSize = "3072m"
```

Comment:
is this the issue because of ubuntu failing all the time?

Reply:
I think so, here is the summary of my investigation https://github.com/Flank/flank/issues/1577